### PR TITLE
here are faqs, first pass #24

### PR DESCRIPTION
**Pull request by drnikko**
reviews wanted. This is a Work In Progress!  We had talked a while about starting this page, so I gathered a few of the most common questions and dropped quick answers in here.

### CODE REVIEW

**Comment by cleverington** [comment]
I think it would also be useful to include the DD&I Booth Talking Points:

https://docs.google.com/document/d/1QdWI9AjvjS42n6jbfk2Vn92xD2zgZPeMxmkuz_s9hTI/edit?usp=sharing

**Comment by YesCT** [comment]
https://github.com/drupaldiversity/administration/issues/38 (Create a slackbot to help moderate dialogue)

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: FAQ
+permalink: /faq/
+---
+
+**Affiliations**
+- _Are you a part of the Drupal Association?_
+      No. We're a collective of volunteers.
+
+- _Didn't you have space in the Drupal Association's booth at DrupalCon Baltimore?_
+Yep. They donated it to us.
```
Comment (douggreen) [inline faq.md]: Is the Yep/Nope tone intentional/playful/helpful or off-putting? I'd just stick with Yes and No, but that's me.

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+**Funding**
+
+We are a collective of volunteers that neither handles money nor has a way to receive or disburse funds.  
+
+- _Who bought the booth space?_
+No one. The Drupal Association donated it.
+
+- _Who paid for the stickers?_
+Digital Echidna, and they asked to do so anonymously.
```
Comment (douggreen) [inline faq.md]: If Digital Echida asked to be anonymous why are you mentioning them now. I suggest either dropping that part of the sentence, or explain further that they've now gone public and given permission to mention them.

**Review by douggreen — COMMENTED** [review]
Where does all this live, is it just on github, or does it power a site? If it lives only on github, please promote a README.md that links to all of the other md.

File: faq.md (hunk 2 above)
Comment (drnikko) [reply faq.md]: Heard on both accounts - clarified the language around this for further discussion.   thanks for commenting!

**Comment by datawench** [comment]
It seems to me that the entirety of this faq is devoted to disclaimers. If I were to read this completely objectively, I would come to the conclusion that DDI amounts to little more than a fan club. If no one speaks for a group, and a group speaks for no one, what is that group's nature and purpose? The fact is that the DA _did_ provide that booth space, which means there _is_ an intent, and a voice. I appreciate the caution, but feel that it only reinforces the impression of something being camouflaged.

~Unless the group's main movers are supremely careful to make sure that group communications remain strictly informational, then the idea there is no group voice can't hold up. And that seems very unlikely.~ 

~And a "group" voice cannot be a cover for the individual voice and personal responsibility. Unless, of course, one is a corporation, in which case all bets are off.~ 

Scratched the last two but left them up, because they're a bit fuzzy and probably not what I would write with another five minutes' thought, but I also own my words.

**Comment by mtift** [comment]
[moved to a different issue]

**Comment by sugaroverflow** [comment]
I added some wording around the twitter account handling in 8fdb8af

**Pull request merged**